packages/utils/src/stakerUtils.ts

```diff
@@ -1,70 +1,41 @@
-import { UserSettingsAllDnps, Network } from "@dappnode/types";
-
-// TODO: remove these utils
-
-/**
- * Get the validator service name.
- * - Nimbus package is monoservice (beacon-validator)
- * - Prysm, Teku, Lighthouse, and Lodestar are multiservice (beacon, validator)
- */
-function getValidatorServiceName(dnpName: string): string {
-  return dnpName.includes("nimbus") ? "beacon-validator" : "validator";
-}
-
-/**
- * Get the beacon service name
- * - Nimbus package is monoservice (beacon-validator)
- * - Prysm, Teku, Lighthouse, and Lodestar are multiservice (beacon, validator)
- */
-export function getBeaconServiceName(dnpName: string): string {
-  return dnpName.includes("nimbus") ? "beacon-validator" : "beacon-chain";
-}
+import { UserSettings, Network } from "@dappnode/types";
 
 /**
  * Get the user settings for the consensus client.
  * It may be different depending if it is multiservice or monoservice and all the envs are
  * set in the same service
  */
-export function getConsensusUserSettings({
-  dnpName,
-  network
-}: {
-  dnpName: string;
-  network: Network;
-}): UserSettingsAllDnps {
-  const validatorServiceName = getValidatorServiceName(dnpName);
-  const beaconServiceName = getBeaconServiceName(dnpName);
+export function getConsensusUserSettings({ network }: { network: Network }): UserSettings {
+  const validatorServiceName = "validator";
+  const beaconServiceName = "beacon-chain";
+  const beaconValidatorServiceName = "beacon-validator";
   const defaultDappnodeGraffiti = "validating_from_DAppNode";
   const defaultFeeRecipient = "0x0000000000000000000000000000000000000000";
   return {
-    [dnpName]: {
-      environment:
-        beaconServiceName === validatorServiceName
-          ? {
-              [validatorServiceName]: {
-                // Fee recipient is set as global env, keep this for backwards compatibility
-                ["FEE_RECIPIENT_ADDRESS"]: defaultFeeRecipient, // TODO: consider setting the MEV fee recipient as the default
-                // Graffiti is a mandatory value
-                ["GRAFFITI"]: defaultDappnodeGraffiti,
-                // Checkpoint sync is an optional value
-                ["CHECKPOINT_SYNC_URL"]: getDefaultCheckpointSync(network)
-              }
-            }
-          : {
-              [validatorServiceName]: {
-                // Fee recipient is set as global env, keep this for backwards compatibility
-                ["FEE_RECIPIENT_ADDRESS"]: defaultFeeRecipient,
-                // Graffiti is a mandatory value
-                ["GRAFFITI"]: defaultDappnodeGraffiti
-              },
+    environment: {
+      // TODO: Remove once Nimbus is split into 2 services
+      [beaconValidatorServiceName]: {
```

Comment:
It is okay to include it in all cases, because the `applyUserSettings` function will only take into account the existing services in the compose of the package to install/start:
![image](https://github.com/user-attachments/assets/5d6b9d22-4b38-4ca9-b5b4-b58e89998d8e)